api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -167,8 +167,11 @@ public void exportWorkspaces(List<Long> workspaceIds) {
               .collect(Collectors.toList());
       if (!rdrWorkspacesList.isEmpty()) {
         rdrApiProvider.get().getApiClient().setDebugging(true);
-        rdrApiProvider.get().exportWorkspaces(rdrWorkspacesList);
-        updateDbRdrExport(RdrEntity.WORKSPACE, workspaceIds);
+        rdrApiProvider.get().exportWorkspaces(rdrWorkspacesList, backfill);
```

Comment:
When backfilling, do not modify the export table

Reply:
Coming back to this, I just found a bug on L172 below. The predicate is incorrect when `backfill=null`.

But also on this comment - why would we not want to modify the export table when backfilling? Seems like we should be doing so. I will send a PR to fix the bug.

Reply:
See https://github.com/all-of-us/workbench/pull/5017